microshift_support/microshift-version.adoc

```diff
@@ -0,0 +1,48 @@
+:_content-type: ASSEMBLY
+[id="microshift-version"]
+= MicroShift Version
+include::_attributes/attributes-microshift.adoc[]
+:context: microshift-version
+toc::[]
+
+[id="microshift-version-cli"]
+== Checking the MicroShift Version from the command line
+
+From the command line, use `microshift version` to check the version information.
+
+[source,terminal]
+----
+$ microshift version
+----
+
```

Comment:
Chaining portions of a step together, including input with output, requires a `+` between the lines:
```suggestion
+
```